File: Api_Qmatic/Entities/Schedule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities
{
    public enum eState { busy, free, clientCanceled, businessCanceled }

    public class Schedule
    {
        public DateTime Date { get; set; }
        public TimeSpan StartHour { get; set; }
        public TimeSpan EndHour { get; set; }
        public eState State { get; set; }
        public string ClientId { get; set; }
    }
}

File: trial/src/app/entities/schedule.ts
import { Time } from "@angular/common";

enum eState { busy, free, clientCanceled, businessCancele };

export const states = {
    0: 'Busy',
    1: 'Free',
    2: 'clientCanceled',
    3: 'businessCanceled',

    busy: 1,
    free: 1,
    clientCanceled: 2,
    businessCanceled: 3
};

export class schedule {
    Date: Date;
    StartHour: Time;
    EndHour: Time;
    State: number;
    ClassState: string;
    clientId: string;
}